catapult-vertx/src/main/kotlin/demo/gos/catapult/CatapultVerticle.kt

```diff
@@ -134,11 +78,20 @@ class Catapult(private val vertx: Vertx, private val id: String, private val x:
   }
 
   private suspend fun update(delta: Double) {
+    makeNoise()
     display()
     val newBoulders = boulders.filter { !it.update(delta) }
     boulders = newBoulders.toMutableList()
   }
 
+  private fun makeNoise() {
```

Comment:
Actually, I think "display" and "makeNoise" could be the same. Not only because we can but also because it makes sense. "display" means emitting photons, and "make noise", well, more or less the same :)